ed448/Cargo.toml

```diff
@@ -0,0 +1,45 @@
+[package]
+authors = ["RustCrypto Developers"]
+categories = ["cryptography"]
+description = """A pure-Rust implementation of Ed448 and Curve448 and Decaf.
+This crate also includes signing and verifying of Ed448 signatures.
+"""
+documentation = "https://docs.rs/ed448-goldilocks"
+exclude = [".gitignore", ".github/*"]
+edition = "2021"
+homepage = "https://docs.rs/ed448-goldilocks/"
+keywords = ["cryptography", "decaf", "ed448", "ed448-goldilocks"]
+license = "BSD-3-Clause"
+name = "ed448"
+readme = "README.md"
+repository = "https://github.com/RustCrypto/elliptic-curves/tree/master/ed448"
+rust-version = "1.85"
+version = "0.17.0-pre.0"
+
+[dependencies]
+crypto-bigint = { version = "0.6.0-rc.3", features = ["hybrid-array"], default-features = false }
+crypto_signature = { version = "2.3.0-pre.6", default-features = false, features = ["digest", "rand_core"], optional = true, package = "signature" }
```

Comment:
if we put this on the pre release for `signature` and `rand@0.9` we will run into the issue that this is going to be unusable as in https://github.com/RustCrypto/signatures/issues/930

Reply:
Let’s get this merged as-is and follow up on that

Reply:
It’s also not the same problem as this crate has existing releases with older deps whereas `ml-dsa`/`slh-dsa` do not

Reply:
fyi I made https://github.com/dignifiedquire/cx448/ based on this code, which I will publish for now, so I can use a more up to date version